tests/TestChannelDataExtractor.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from enum import Enum
+
+from Commons.ChannelsID import ChannelsID
+from Extract.ChannelDataExtractor import ChannelDataExtractor
+
+
+class TestChannelDataExtractor(unittest.TestCase):
+    channel_id: Enum
+    extract_data: dict
+
+    def setUp(self) -> None:
+        self.channel_id = ChannelsID("channels.txt").get_channels_id()
+        self.extract_data = ChannelDataExtractor().extract(self.channel_id)
+
+    def test_extract(self) -> None:
```

Comment:
in unit tests naming there is a necessity to give for each method the name of narrow case which will be tested.